### PR TITLE
Number-format Changes

### PR DESCRIPTION
**Pull request by Ippezrobert**

### CODE REVIEW

**Comment by rajuan** [comment]
can you please provide the jira issue for this fix

**Comment by Ippezrobert** [comment]
PR is for issue https://mifosforge.jira.com/browse/MIFOSX-2670

File: app/views/groups/transferclients.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
                 <div class="col-sm-3 col-md-3">
                     <label class="control-label col-sm-12 center">{{ 'label.input.selectedclients' | translate }}</label>
                     <select multiple ng-model="selectedClients" class="form-control clear">
-                        <option ng-repeat="member in selectedMembers" value="{{member.id}}">{{member.accountNo}}{{member.displayName}}</option>
+                        <option ng-repeat="member in selectedMembers" value="{{member.id}}">{{member.accountNo}} | {{member.displayName}}</option>
```
Comment (nazeer1100126) [inline app/views/groups/transferclients.html]: Why this change is required? 
If you want to add '|' between accountNo and displayName for selected members,
you should be doing same thing even for available clients.

**Comment by Ippezrobert** [comment]
@nazeer1100126 , Please check the new PR i have sent, hence closing this PR
